packages/tap-runner/test/integration/tap-test-runner.it.spec.ts

```diff
@@ -0,0 +1,132 @@
+import os from 'os';
+
+import { expect } from 'chai';
+import { factory, TempTestDirectorySandbox, testInjector, assertions } from '@stryker-mutator/test-helpers';
+import { DryRunStatus } from '@stryker-mutator/api/test-runner';
+
+import { TapTestRunner } from '../../src/index.js';
+import { createTapTestRunnerFactory } from '../../src/tap-test-runner.js';
+import { findTestyLookingFiles } from '../../src/tap-helper.js';
+import { TapRunnerOptionsWithStrykerOptions } from '../../src/tap-runner-options-with-stryker-options.js';
+import { defaultTestFilesGlob } from '../helpers/tap-test-runner-constants.js';
+
+describe('Running in an example project', () => {
+  let sut: TapTestRunner;
+  let sandbox: TempTestDirectorySandbox;
+  let testFilter: string[] = [];
+
+  beforeEach(async () => {
+    sandbox = new TempTestDirectorySandbox('example');
+    await sandbox.init();
+    (testInjector.options as TapRunnerOptionsWithStrykerOptions).tap = {
+      testFiles: defaultTestFilesGlob,
+    };
+    sut = testInjector.injector.injectFunction(createTapTestRunnerFactory('__stryker2__'));
+    await sut.init();
+
+    const excludeFiles = ['tests/bail.spec.js', 'tests/error.spec.js'];
+    testFilter = (await findTestyLookingFiles(defaultTestFilesGlob)).filter((file) => !excludeFiles.includes(file));
```

Comment:
This is a common use case. I think we should allow users to configure exclusions.